parsers/abc.c

```diff
@@ -0,0 +1,124 @@
+/*
+*   Copyright (c) 2009, Eric Forgeot
+*
+*   Based on work by Jon Strait
+*
+*   This source code is released for free distribution under the terms of the
+*   GNU General Public License.
+*
+*   This module contains functions for generating tags for Abc files.
+*/
+
+/*
+*   INCLUDE FILES
+*/
+#include "general.h"	/* must always come first */
+
+#include <ctype.h>
+#include <string.h>
+
+#include "parse.h"
+#include "read.h"
+#include "vstring.h"
+#include "routines.h"
+#include "entry.h"
+
+/*
+*   DATA DEFINITIONS
+*/
+
+static kindDefinition AbcKinds[] = {
```

Comment:
If 's' is not used, let's remove it.
Event there is only one kind, I would like you to define enum for kinds like:
```
enum AbcKind {
   K_MEMBER,
};
```

Reply:
I've removed all the commented-out code from the parser which is unmaintained and there doesn't seem to be anything of a great value. I've also made some cleanups in the parser.